### PR TITLE
Create sidebar and navbar.

### PR DESCRIPTION
**Pull request by gonzalo-plaza**

### CODE REVIEW

**Comment by gonzalo-plaza** [comment]
@PVillasana @pedrojgm @Andresws12 @JuanJosePG Please, review the code to can know if I am doing a correct work with the code.

File: src/components/SideBar/SideBar.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="side-bar">
+    <el-row>
```
Comment (Andresws12) [inline src/components/SideBar/SideBar.vue]: @gonzalo-plaza I think that this row and column is not necessary

File: src/main.ts [hunk 2]
```diff
@@ -3,8 +3,12 @@ import App from "./App.vue";
 import "./registerServiceWorker";
 import router from "./router";
 import store from "./store";
+import Element from "element-ui";
```
Comment (Andresws12) [inline src/main.ts:8]: @gonzalo-plaza Please separate with white line the libraries, ts imports and differents things

File: src/components/SideBar/SideBar.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        </el-menu>
+      </el-col>
+    </el-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+@Component({})
```
Comment (Andresws12) [inline src/components/SideBar/SideBar.vue]: @gonzalo-plaza please, add the name component on all components

File: src/components/NavBar/NavBar.vue [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    </el-header>
+  </el-container>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+import SideBar from "@/components/SideBar/SideBar.vue";
+
+@Component({
```
Comment (Andresws12) [inline src/components/NavBar/NavBar.vue:29]: @gonzalo-plaza The name componente, please

**Comment by Andresws12** [comment]
@gonzalo-plaza I don't like the name project.... Maybe is redundant, I would say...  OpenSlot-Guide-CSS.  @PVillasana @pedrojgm @JuanJosePG  what do you think?

**Comment by gonzalo-plaza** [comment]
@Andresws12 @PVillasana @pedrojgm @JuanJosePG Please review the code, I need merge this to can use the subdomains and can view the changes easier.

**Comment by Andresws12** [comment]
@gonzalo-plaza Remember, resolve the conversation and answer if it is solved this question.
1º. The logo. The quality from the logo is low.... please change to svg and the height is so big, please change to more small, like the screenshot
2º. I don't like this box-shadow, please remove o change to other more smooth
3º. The title, maybe is so thick, i would say change the font-weight to 500
4º. The project title? for now change it to Open Slot-Guide-CSS, the title, name project, remember change on all parts
5º. Why you don't use scss variables?
image

File: src/components/SideBar/SideBar.vue (hunk 1 above)
Comment (gonzalo-plaza) [reply src/components/SideBar/SideBar.vue]: @Andresws12 Can I resolve this conversation?

File: src/main.ts (hunk 2 above)
Comment (gonzalo-plaza) [reply src/main.ts:8]: @Andresws12 Can I resolve this conversation?

File: src/components/SideBar/SideBar.vue (hunk 3 above)
Comment (gonzalo-plaza) [reply src/components/SideBar/SideBar.vue]: @Andresws12 Can I resolve this conversation?

File: src/components/NavBar/NavBar.vue (hunk 4 above)
Comment (gonzalo-plaza) [reply src/components/NavBar/NavBar.vue:29]: @Andresws12 Can I resolve this conversation?

**Comment by gonzalo-plaza** [comment]
@Andresws12 I have fix the things that you have comment me, please, review this and I need know if the solution with the url of the Lexend font is correct, because is working but I don´t know if the solution is the better way.

**Comment by gonzalo-plaza** [comment]
@Andresws12 @PVillasana @pedrojgm @JuanJosePG Please, review the code

File: src/components/NavBar/NavBar.vue [hunk 5]
```diff
@@ -0,0 +1,55 @@
+<template>
+  <el-container class="container">
+    <el-header class="container_header">
+      <el-row type="flex" justify="space-between">
+        <el-col :span="0.5">
+          <img
+            class="container_header_img"
+            src="../../assets/img/LogoOSS.svg"
```
Comment (Andresws12) [inline src/components/NavBar/NavBar.vue:8]: @gonzalo-plaza I can't see the logo. is only on my pc?
image

File: src/components/NavBar/NavBar.vue [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+          />
+        </el-col>
+        <el-col :span="8">
+          <span class="container_header_title">OpenSlot-Guide-CSS</span>
+        </el-col>
+        <el-col :span="0.5">
+          <nav-bar-language />
```
Comment (Andresws12) [inline src/components/NavBar/NavBar.vue]: @gonzalo-plaza Please, add all translations to can  test the translations.

File: src/components/NavBar/NavBar.vue [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    &_title {
+      line-height: 60px;
+      color: $secondary-color-light;
+      font-weight: 500;
+      font-size: 20px;
+    }
+
+    &_img {
+      height: 35px;
+      padding-top: 12px;
```
Comment (Andresws12) [inline src/components/NavBar/NavBar.vue]: @gonzalo-plaza Remember use rem

File: src/components/NavBar/NavBar.vue (hunk 5 above)
Comment (Andresws12) [reply src/components/NavBar/NavBar.vue:8]: and remove ..png, image,   please

**Comment by gonzalo-plaza** [comment]
@Andresws12 @PVillasana @JuanJosePG @pedrojgm Please, review the code.

File: src/views/About.vue [hunk 8]
```diff
@@ -1,7 +1,5 @@
 <template>
-  <div class="about">
-    <h1>This is an about page</h1>
-  </div>
+  <div class="about"></div>
```
Comment (JuanJosePG) [inline src/views/About.vue]: If the div is empty, you don't need to close it with "< /div >". Do it like you did in Home.vue

File: src/components/NavBar/NavBar.vue (hunk 5 above)
Comment (gonzalo-plaza) [reply src/components/NavBar/NavBar.vue:8]: @Andresws12 This is solved

File: src/components/NavBar/NavBar.vue (hunk 6 above)
Comment (gonzalo-plaza) [reply src/components/NavBar/NavBar.vue]: @Andresws12 This is solved

File: src/components/NavBar/NavBar.vue (hunk 7 above)
Comment (gonzalo-plaza) [reply src/components/NavBar/NavBar.vue]: @Andresws12 This has been apply

File: src/views/About.vue (hunk 8 above)
Comment (gonzalo-plaza) [reply src/views/About.vue]: @JuanJosePG This is solved

**Comment by gonzalo-plaza** [comment]
@pedrojgm @Andresws12 @JuanJosePG @PVillasana This is okay to can be reviewed.

File: src/components/SideBar/SideBar.vue [hunk 9]
```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="side-bar">
+    <el-menu
+      class="side-bar_menu"
+      default-active="/"
+      background-color="#D4E2EA"
```
Comment (Andresws12) [inline src/components/SideBar/SideBar.vue:6]: @gonzalo-plaza Maybe on the future MR, you can use a global enum to define the app colors and use on the application

Comment (Andresws12) [reply src/components/SideBar/SideBar.vue:6]: For me is okay to merge

Comment (gonzalo-plaza) [reply src/components/SideBar/SideBar.vue:6]: Okay, in the next MR, I apply a global enum. Thanks!

**Comment by pablovillasana** [comment]
@gonzalo-plaza Please change some directories name where this naming is used, and consider using camelCase for directories. (I think that you will only have to rename the components sub-directories, but please check if it is used on another place and replace it aswell)

image

**Comment by Andresws12** [comment]
@gonzalo-plaza Please review the checks failed

File: src/App.vue [hunk 10]
```diff
@@ -1,32 +1,50 @@
...
+      </el-col>
+    </el-row>
   </div>
 </template>
+<script lang="ts">
+import { Component, Vue, Watch } from "vue-property-decorator";
 
+import mainStore from "@/store/main-store/MainStore";
+
+import NavBar from "@/components/NavBar/Navbar.vue";
```
Comment (Andresws12) [inline src/App.vue]: @gonzalo-plaza `import NavBar from "@/components/NavBar/NavBar.vue";` B**

File: src/App.vue [hunk 11]
```diff
@@ -1,32 +1,50 @@
...
+import SideBar from "@/components/SideBar/Sidebar.vue";
```
Comment (Andresws12) [inline src/App.vue]: @gonzalo-plaza `import SideBar from "@/components/SideBar/SideBar.vue";` B**

File: src/App.vue (hunk 10 above)
Comment (gonzalo-plaza) [reply src/App.vue]: @Andresws12 Okay, I have understand the problem. Thank you!!

File: src/App.vue (hunk 11 above)
Comment (gonzalo-plaza) [reply src/App.vue]: @Andresws12 Thank for your help. I have understand the problem.

**Comment by gonzalo-plaza** [comment]
@Andresws12 I have look the solution of the problem with the import of components. This is ready to merge?

**Pull request merged**